src/ao.ts

```diff
@@ -0,0 +1,46 @@
+import { SMA } from './sma';
+
+/**
+ Awesome Oscillator (AO) is an indicator that is non-limiting oscillator,
+ providing insight into the weakness or the strength of a stock.
+ The Awesome Oscillator is used to measure market momentum and
+ to affirm trends or to anticipate possible reversals.
+ */
+
+export class AO {
+    private smaSlow: SMA;
+    private smaFast: SMA;
+    private smaSlowValue: number;
+    private smaFastValue: number;
+
+    constructor() {
+        this.smaSlow = new SMA(34);
```

Comment:
is it always static?

Reply:
its meant to be used with 34 and 5, but i'll add custom periods in next commit